.changeset/bright-ducks-battle.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
the name of the file is weird

Reply:
Auto generated actually. And will be auto deleted once merged to main branch